sunpy/map/mapbase.py

```diff
@@ -418,7 +418,11 @@ def _get_lon_lat(self, frame):
         r = frame.represent_as(UnitSphericalRepresentation)
         return r.lon.to(self.spatial_units[0]), r.lat.to(self.spatial_units[1])
 
+    def _meta_hash(self):
```

Comment:
I was anticipating this:
```suggestion
    @property
    def _meta_hash(self):
```